news/2021-03-08-skinning-contest-announcement.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Skinning Contest: Announcement"
+date: 2021-03-08 12:00:00 +0000
+---
+
+Skinners, this one is for you! Create your finest skin! Why? Because it is your time to shine!
+
+![](/wiki/shared/news/2021-03-08-skinning-contest-announcement/skinning_contest_banner_announcement.jpg)
+
+Skinning has been one of the most notable features of osu! since forever. Whether it be an elegantly designed anime-themed skin, or a super simple and minimalistic one, everyone has their own favourite. However, the very creators of these masterpieces still remain unknown by many people. Therefore, now it is time for the skinners extraordinaire to come forth and show off their skills!
+
+Hosted by one of the largest skinning communities, [skinship](https://skinship.xyz/), the contest is apt for both experienced and inexperienced skinners, with no theme restrictions whatsoever. In other words, everyone is allowed to join with a skin of any theme!
+
+Are you interested in joining but new to skinning? Fear not! Nearly everything you need to know is available at our [skinning tutorial](https://skinship.xyz/tutorial/introduction). If you need further assistance, feel free to visit the [skinning forums](https://osu.ppy.sh/community/forums/15) and consult other fellows!
+
+## Participation
+
+If you're reading this, you're probably interested in participating too. So let's dive into it!
+
+In order to participate in the contest, you will have to create a brand-new skin all by yourself, ready to compete against skin creators from around the globe!
+
+In addition to the skin, you need to prepare a forum thread accommodating it. That's right, you can't just submit your skin file and run away, but also need to present your skin in a forum post however you like. Said forum thread must be located under the [Skinning WIP subforum](https://osu.ppy.sh/community/forums/119).
+
+Since all entries will have their respective forum threads publicly available, contestants and their submissions will not be anonymized. In fact, we encourage you to show off your work as much as you wish!
+
+## Voting
+
+After the submission phase is over, the entries will be made available for community voting.
+
+The results of the community voting will determine the winner of the contest. Entry with the most votes wins!
+
+## Schedule
+
+"When?" you say? Right here and right now! The dates will become clearer, just pay attention to the following.
+
+- Submission: **4** weeks starting from the release of this news post
+- Voting: **2** weeks starting from a separate news post
+
+## Rules
+
+*Note: this is a short summary of the rules, for further details please visit this [forum post](https://osu.ppy.sh/community/forums/topics/1266342).*
```

Comment:
The rules should *all* be listed on the wiki page. A forum thread should not be used at all. I don't think it should even be linked in this article.

Also this note is only present here, not on the contest's wiki page, making it confusing as to whether it's even required.

Reply:
They are all listed on here, guess this is a leftover from early planning stages where no wiki article was planned. Removed them from the post as well as this note